properties/properties.go

```diff
@@ -84,6 +85,10 @@ func (p *Properties) Update(props map[string]string) {
 
 	for k, v := range props {
 		p.m[k] = v
+
+		if k == "global.cert-manager.expiry" {
```

Comment:
I think this can be a property listener in is-healthy directly  `health.cert-manager.expiryGracePeriod`